core/src/martian/schema.cljc

```diff
@@ -98,7 +98,13 @@
 
 (defn wrap-default [{:keys [default]} schema]
   (if (some? default)
-    (st/default schema default)
+    (let [default
+          ;patch for `inf` as a default value
+          (if (and (= schema s/Int) (= default "inf"))
+            #?(:clj Long/MAX_VALUE
+               :cljs (Math/pow 10 1000))
```

Comment:
Is this the best that can be done on cljs?

Reply:
No. In fact, I realized the cljs test failed, but I have trouble to run the tests locally. Do you know how do I specify the correct Chrome location to run the figwheel tests?

Reply:
You can set the path to chrome here: https://github.com/oliyh/martian/blob/master/core/test.cljs.edn#L5